dataflow/build.gradle

```diff
@@ -66,6 +66,7 @@ task manual(group: 'Documentation') {
 }
 
 tasks.withType(Test) {
+    enabled = false
```

Comment:
Because gradlew test automatically use junit test framework, we set this to false will remove the warnings.
And the actual test for dataflow is `allDataFlowTests` which is still running.